src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackContextBlock.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Notifier\Bridge\Slack\Block;
+
+final class SlackContextBlock extends AbstractSlackBlock
+{
+    public function __construct()
+    {
+        $this->options['type'] = 'context';
+    }
+
+    /**
+     * @return $this
+     */
+    public function text(string $text, bool $markdown = true, bool $emoji = true, bool $verbatim = false): self
+    {
+        if (10 === \count($this->options['elements'] ?? [])) {
```

Comment:
given that both images and texts count for the number of elements, I'd use a constant for the limit.